ethportal-api/src/types/portal_wire.rs

```diff
@@ -590,21 +611,20 @@ impl From<Accept> for Value {
 #[allow(clippy::unwrap_used)]
 mod test {
     use super::*;
-    use crate::utils::bytes::hex_encode_upper;
+    use std::str::FromStr;
     use test_log::test;
 
     #[test]
-    #[should_panic]
     fn protocol_id_invalid() {
         let hex = "0x500F";
-        ProtocolId::from_str(hex).unwrap();
+        assert!(!MAINNET.portal_networks.contains_right(hex));
     }
 
     #[test]
     fn protocol_id_encoding() {
         let hex = "0x500A";
-        let protocol_id = ProtocolId::from_str(hex).unwrap();
-        let expected_hex = hex_encode_upper(Vec::try_from(protocol_id).unwrap());
+        let protocol_id = MAINNET.portal_networks.get_by_right(hex).unwrap();
```

Comment:
I'm not a huge fan of using the native `BiHashMap` api. It's not obvious (without looking up the map &/or `BiHashMap` source code), what these functions do (aka left vs. right)... Could you `impl` something more intuitive, like `get_protocol_id` / `get_protocol_id_hex` (or whatever fn name makes the most sense here)?